src/System.Net.Sockets/src/System/Net/Sockets/BaseOverlappedAsyncResult.Unix.cs

```diff
@@ -23,7 +23,7 @@ public BaseOverlappedAsyncResult(Socket socket, Object asyncState, AsyncCallback
             if (NetEventSource.IsEnabled) NetEventSource.Info(this, socket);
         }
 
-        protected void CompletionCallback(int numBytes, SocketError errorCode)
+        internal void CompletionCallback(int numBytes, SocketError errorCode)
```

Comment:
Ok.  I'm going to merge the change and then I'll fix this as part of the SendPacketsAsync change.